fullscreen/model/move-to-fullscreen-iframe-manual.html

```diff
@@ -22,10 +22,13 @@
     document.onfullscreenchange = t.step_func_done(() => {
       assert_equals(document.fullscreenElement, null, "document's fullscreen element");
```

Comment:
Probably consider adding assertions in the first level of event handler to clarify that both documents have fullscreen element at that point.

Reply:
Done!